### ISSUE TITLE
Viewport only takes a quarter of the available space in MacOS

### ISSUE THREAD

**Issue by cavenel** [AICAN-Research/FAST-Pathology#87]
When opening an image in FastPathology in MacOS, the image is only visible in a quarter of the usual space:
![image](https://github.com/AICAN-Research/FAST-Pathology/assets/6164758/404db8b0-8584-4a00-9f75-c3fc55150283)

When zooming on the tissue, it zooms on the point that would be there if the image was taking the full size, and not the actual visible point.
Resizing the main window actually fixes the bug, until next restart.
**System:**
- OS and version: MacOS 12.6.8

Could be an issue with MacOS big resolution adding a "device pixel ratio" of 2?

**Comment by andreped** [comment]
Yes, this is a known bug on macOS. Resizing does indeed fix the issue, but of course it would be better if this just worked out-of-the-box, @smistad.

**Comment by cavenel** [comment]
Ah ok I didn't find the issue so I thought I could open my own :p
Here is a small video of this behavior:
![Kapture 2023-09-04 at 15 36 22](https://github.com/AICAN-Research/FAST-Pathology/assets/6164758/f1911643-f9b9-4abc-a5d2-4ac0d226fdde)

**Comment by andreped** [comment]
I don't think we made an issue for it, so it is great that you did, @cavenel!
